### DISCUSSION TITLE
Scrolling line instead of + for "new tab"

### DISCUSSION THREAD

**Discussion by AlexRydzewski** [piroor/treestyletab#3659] [Q&A]
Recently, I noticed a scrolling line where there used to be a plus symbol for opening a new tab. It's annoying. Is there a way to disable it, does anyone know?

**Comment by piroor** [comment]
Do you use any user stylesheet? And could you paste a screenshot of the sidebar with scroll line?

**Comment by SHHSSH** [comment]
> Recently, I noticed a scrolling line where there used to be a plus symbol for opening a new tab. It's annoying. Is there a way to disable it, does anyone know?

Do you want to remove the + at the bottom of TST?
If so add this to your css down the bottom of TST's settings.

/* Hide 'new tab' button. */
.newtab-button {
  display: none;
}
